ghost/collections/src/PostsDataRepositoryBookshelf.ts

```diff
@@ -0,0 +1,17 @@
+type PostsDataRepositoryBookshelfDeps = {
+    Post: any;
+}
+
+export class PostsDataRepositoryBookshelf {
```

Comment:
We want to keep all bookshelf dependent code together in the Ghost core - so we shouldn't define this repository in the package

Reply:
Hm ok. So the packages would only hold the "in memory" kind of repositories and we inject any other Bookshelf/Redis/whatevs from the outside. Am I understanding it correctly?

Reply:
Yeah - ideally we keep all the bookshelf related code in one package so it's easier to rip out down the line.

The idea is that the package contains the business logic, and the infrastructure implementations can be outside of it - this keeps the scope of the package narrower and doesn't require stubbed bookshelf dependencies for testing!

At the moment bookshelf repositories have been implemented in `Ghost/ghost/core/server/services/**/BlahBookshelfRepository` but we'll probably want to pull those out to `Ghost/ghost/core/server/repositories/bookshelf` or `Ghost/bookshelf-repositories` or something to keep them all together

Reply:
yup, makes sense. I've moved the bookshelf posts wrapper into core's codebase 👍